VEDA_OS01/management/commands/tests/test_re_encrypt_transcript_credentials.py

```diff
@@ -0,0 +1,112 @@
+"""
+Tests of the re_encrypt_transcript_credentials management command.
+"""
+from django.conf import settings
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import TestCase, override_settings
+
+from cryptography.fernet import InvalidToken
+from mock import patch
+
+
+from VEDA_OS01.models import TranscriptCredentials, TranscriptProvider
+from VEDA_OS01.tests.utils import invalidate_fernet_cached_properties
+
+
+OLD_FERNET_KEYS_LIST = ['test-ferent-key']
+
+
+class ReEncryptTranscriptCredentialsTests(TestCase):
+    """
+    Management command test class.
+    """
+
+    def setUp(self):
+        """
+        Test setup.
+        """
+        self.credentials_data = {
+            'org': 'MAx',
+            'provider': TranscriptProvider.THREE_PLAY,
+            'api_key': 'test-key',
+            'api_secret': 'test-secret'
+        }
+        TranscriptCredentials.objects.create(**self.credentials_data)
+
+    def tearDown(self):
+        """
+        Test teardown.
+        """
+        # Invalidate here so that every new test would have FERNET KEYS from tests.py initially.
+        invalidate_fernet_cached_properties()
+
+    def verify_access_credentials(self):
+        # Fetch a record to check if we are able to get encrypted data.
+        # Accessing object that is not able to be decrypted, would throw InvalidToken error.
+        transcript_credentials = TranscriptCredentials.objects.get(
+            org=self.credentials_data['org'], provider=self.credentials_data['provider']
+        )
+        self.assertIsNotNone(transcript_credentials)
```

Comment:
`transcript_credentials` will never be `None` because `TranscriptCredentials.objects.get` will raise `DoesNotExist` if record not found.

I think `assertIsNotNone` is not needed here.